upstream-community-operators/microcks/0.2.1/microcks-operator.v0.2.1.clusterserviceversion.yaml

```
@@ -0,0 +1,242 @@
+    createdAt: 2019-11-04T09:01:00Z
+    support: microcks.github.io
+    repository: https://github.com/microcks/microcks-ansible-operator
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "microcks.github.io/v1alpha1",
+          "kind": "MicrocksInstall",
+          "metadata": {
+            "name": "my-microcksinstall"
+          },
+          "spec": {
+            "name": "my-microcksinstall",
+            "version": "0.7.1",
+            "microcks": {
+              "replicas": 2
+            },
+            "postman": {
+              "replicas": 2
+            },
+            "keycloak": {
+              "install": true,
+              "persistent": true,
+              "volumeSize": "1Gi"
+            },
+            "mongodb": {
+              "install": true,
+              "persistent": true,
+              "volumeSize": "2Gi"
+            }
+          }
+        },
+        {
+          "apiVersion": "microcks.github.io/v1alpha1",
+          "kind": "MicrocksInstall",
+          "metadata": {
+            "name": "my-microcksinstall-minikube"
+          },
+          "spec": {
+            "name": "my-microcksinstall-minikube",
+            "version": "0.7.1",
+            "microcks": {
+              "replicas": 2,
+              "url": "microcks.192.168.99.100.nip.io"
+            },
+            "postman": {
+              "replicas": 2
+            },
+            "keycloak": {
+              "install": true,
+              "persistent": true,
+              "volumeSize": "1Gi",
+              "url": "keycloak.192.168.99.100.nip.io"
+            },
+            "mongodb": {
+              "install": true,
+              "persistent": true,
+              "volumeSize": "2Gi"
+            }
+          }
+        }
+      ]
+spec:
+  displayName: Microcks Operator
+  version: 0.2.1
+  description: >
+    Microcks is an open source project those goal is to provide a platform for referencing, deploying mocks and allow contract-testing of Services and APIs. It can also be considered as a Service Virtualization solution because it will allow you to provide fake API or Service implementation before development being actually done. It supports both REST API and SOAP WebServices and perfectly integrates into an iterative,contract-first delivery process.
+    
+    Microcks tries not reinventing the wheel and let you capitalize on already known tools of standards. It supports OpenAPI Specification 3.x out-of-the-box and can also relies on popular tools like [SoapUI](http://www.soapui.com) or [Postman](http://www.getpostman.io) to edit your API request / response samples as well as your test scripts.
+    
+    * **Create/Destroy**: Easily launch Microcks installations for your kubernetes namespace.
+  keywords: ['mock', 'test', 'API', 'microservice']
+  maintainers:
+  - name: lbroudoux
+    email: laurent.broudoux@gmail.com
+  provider:
+    name: Microcks.io
+  replaces: microcks-operator.v0.2.1
```

```suggestion
  replaces: microcks-operator.v0.2.0
```

This leads to a endless loop when putting this into a catalog. That's why the test is failing.